app/views/products/popular.html.erb

```diff
@@ -0,0 +1,19 @@
+<h2>Dummy Popular Products Page</h2>
+
+<center>
+  <div class="row">
+    <% @products.each do |product| %>
+      <div class="col-sm-6 col-md-3">
+        <div class="thumbnail">
+          <%= link_to (image_tag product.image.url(:thumb), class: 'image'), product %>
+          <div class="caption">
+            <h3><%= product.name %></h3>
+            <h4>Category: <%= product.category %></h4>
+            <p><%= product.description %></p>
+            <p><%= link_to "Read More", product_path(product), class: "btn btn-default", role: "button" %></p>
+          </div>
+        </div>
+      </div>
+    <% end %>
+  </div>
+</center>
```

Comment:
@cchanningallen Sorry for the confusion! This is just a placeholder page (copied exactly from the products index view) so that I could test the active state functionality on something other than "Home" in the navbar.